testutils/gke/client.go

```diff
@@ -105,8 +106,19 @@ func (gsc *sdkClient) GetCluster(project, region, zone, clusterName string) (*co
 	if zone != "" {
 		return gsc.Projects.Zones.Clusters.Get(project, location, clusterName).Context(context.Background()).Do()
 	}
-	clusterFullPath := fmt.Sprintf("projects/%s/locations/%s/clusters/%s", project, location, clusterName)
-	return gsc.Projects.Locations.Clusters.Get(clusterFullPath).Context(context.Background()).Do()
+	opsFullPath := fmt.Sprintf("projects/%s/locations/%s/clusters/%s", project, location, clusterName)
+	return gsc.Projects.Locations.Clusters.Get(opsFullPath).Context(context.Background()).Do()
+}
+
+// ListClustersInProject lists all the GKE clusters created in the given project.
+func (gsc *sdkClient) ListClustersInProject(project string) ([]*container.Cluster, error) {
+	var clusters []*container.Cluster
+	projectPath := fmt.Sprintf("projects/%s/locations/-", project)
+	resp, err := gsc.Projects.Locations.Clusters.List(projectPath).Do()
```

Comment:
so you are only listing regional clusters? Or this API returns both regional and zonal clusters?

Reply:
I've tested it, it can also return zonal clusters.

Reply:
seems like the GKE API is a little messed up

Reply:
Ah yeah that's true. But only this path uses `-` to represent all `region`s and `zone`s, I guess  it's fine :)